kitsune/sumo/management/commands/enqueue_lag_monitor_task.py

```diff
@@ -9,4 +9,4 @@ class Command(BaseCommand):
     help = "Fire a task that measures the queue lag."
 
     def handle(self, **options):
-        measure_queue_lag.delay(datetime.now())
+        measure_queue_lag.delay(datetime.now().isoformat())
```

Comment:
My understanding is that the JSON serializer that Celery uses doesn't handle `datetime` objects by default, and since the `datetime.now()` function returns a `datetime` object (which would require `pickle` serialization or a custom JSON encoder/decoder), I instead converted the datetime object to a string in `ISO` format.